server/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.8-alpine
+
+EXPOSE 5501
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+
+### TODO: This should be in entrypoint.sh
+ENV PGPASSWORD="do-not-use-in-production"
+
+# "No working compiler found" solved
+RUN apk add --no-cache --virtual .py_deps build-base python3-dev libffi-dev
```

Comment:
Maybe include an `apk update`/`--update` on all of these?

Reply:
can't hurt, but it will take longer everytime when you build the container (which i have done a lot when developing :D )